import_export_celery/tasks.py

```diff
@@ -188,44 +188,58 @@ def run_import_job(pk, dry_run=True):
         return
 
 
-@task(bind=False)
+@task(bind=False,soft_time_limit=36000)
```

Comment:
Why add the time limit?

Reply:
For exporting millions of rows, the celery job takes near about 45 min.
so I have to increase softtimelimt =1hr. otherwise you will get SoftTimeLimit Exception